deploy/crds/keycloak.org_keycloakrealms_crd.yaml

```diff
@@ -160,6 +160,54 @@ spec:
                     - alias
                     type: object
                   type: array
+                clientScopes:
+                  description: Client scopes
+                  items:
+                    description: https://www.keycloak.org/docs-api/10.0/rest-api/index.html#_clientscoperepresentation
```

Comment:
We probably should avoid this kind of descriptions (although I think I missed this in the previous PR I reviewed). This either should be empty or have some meaningful description.

Reply:
Sorry, that was intented as a comment in the source, not CRD docs.

I think the user docs should be exported from keycloak somehow.

For now we can adapt this text I guess: https://www.keycloak.org/docs/latest/server_admin/#core-concepts-and-terms

Reply:
We have a documentation for our REST resources here: https://www.keycloak.org/docs-api/11.0/rest-api/index.html#_realmrepresentation (you may find other resources, such as `ClientRepresentations` or `UserRepresentation` there as well).

Reply:
I'll just leave it empty as you suggest:
* I don't think the current overall CRD documentation is helpful enough overall currently for anyone to use it alone
* I think all "hand written" comments would be removed (and come from OpenAPI) when https://issues.redhat.com/browse/KEYCLOAK-14321 is fixed, so it would not have long time value to work on these files.